package.json

```diff
@@ -14,57 +14,75 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "npm-run-all lint:* test:*",
+    "test": "concurrently \"npm:lint\" \"npm:test:*\" --names \"lint,test:\"",
     "test:ember": "ember test",
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
     "@ember/test-waiters": "^3.0.1",
-    "ember-cli-babel": "^7.22.1",
-    "ember-cli-htmlbars": "^5.3.1",
-    "ember-modifier": "^2.1.1"
+    "ember-auto-import": "^2.6.1",
+    "ember-cli-babel": "^7.26.11",
+    "ember-cli-htmlbars": "^6.2.0",
+    "ember-modifier": "^3.0.0 || ^4.0.0"
```

Comment:
It's best to support both v3 and v4 for now, because many apps are using modifiers that don't work in v4, so they need to stay on v3 until they fix their own modifiers.